lib/ts/recipe/thirdparty/index.ts

```diff
@@ -133,6 +134,9 @@ export default class Wrapper {
     static ThirdPartyAuth = ThirdPartyAuth;
     static SignInAndUp = (prop?: any) => ThirdParty.getInstanceOrThrow().getFeatureComponent("signinup", prop);
     static SignInAndUpTheme = SignInAndUpTheme;
+    static SignInAndUpCallback = (prop?: any) =>
+        ThirdParty.getInstanceOrThrow().getFeatureComponent("signinupcallback", prop);
+    static SignInAndUpCallbackTheme = SignInAndUpCallbackTheme;
```

Comment:
This part is already done by mihaly in his previous PR. Maybe update your PR with the base?

Reply:
As discussed I will do this after this PR is merged to make conflict resolution easier